markdown/events/2023/hackathon-march-2023/uk-google.md

```diff
@@ -32,33 +32,36 @@ Please use the [main registration form](https://nf-co.re/events/2023/hackathon-m
             </thead>
             <tbody>
             <tr>
-                <td data-timestamp="1679905800" data-timeformat="HH:mm z">09:30</td>
+                <td data-timestamp="1679905800" data-timeformat="HH:mm z">08:30</td>
```

Comment:
```suggestion
                <td data-timestamp="1679904000" data-timeformat="HH:mm z">08:30</td>
```
You need to update the timestamps. the other string is overwritten based on the timestamp (to always show local time)

Reply:
Not sure with GMT->BST, but I think I had the right time